osquery/utils/conversions/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ function(generateOsqueryUtilsConversions)
 
   target_link_libraries(osquery_utils_conversions PUBLIC
     osquery_cxx_settings
+    osquery_filesystem
```

Comment:
We might have to look into `filesystem` and see if its dependencies are overly-broad (this could be a rabbit hole).

Reply:
You'll need to remove this to fix the Linux build errors, don't link `osquery_filesystem`.